src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 6;
+    public static final int MAX_SIDE_LENGTH = 101;
+    public static final int DEFAULT_RADIUS = 10;
+    public static final Color DEFAULT_COLOR = Color.WHITE;
+
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
If it constant, you need write it like constant or you can use like fields and better init it in constructor